confdb/aclocal_modules.m4

```diff
@@ -139,6 +139,8 @@ AC_DEFUN([PAC_CONFIG_HWLOC],[
         if test -n "$hwloc_darwin_ldflags" ; then
             echo "hwloc_darwin_ldflags = $hwloc_darwin_ldflags"
             PAC_APPEND_FLAG([$hwloc_darwin_ldflags], [LDFLAGS])
+            # we also need these flags for static linking in the wrapper scripts
+            PAC_APPEND_FLAG([$hwloc_darwin_ldflags], [SAVE_LDFLAGS])
```

Comment:
This version works in manual testing. The naming of `SAVE_LDFLAGS` seems awkward now that we are manipulating it, but functionally it is correct.

Reply:
I see. Yeah, modifying `SAVE_LDFLAGS` is weird. How about we add `@WRAPER_DEPENDNENCY_LDFLAGS@`?

Reply:
Since this is literally the only use for `SAVE_LDFLAGS`, should we just rename it?

Reply:
Sure. That's a better idea.